storageapi/src/vespa/storageapi/message/bucket.cpp

```diff
@@ -552,7 +548,7 @@ NotifyBucketChangeCommand::print(std::ostream& out, bool verbose,
                                  const std::string& indent) const
 {
     out << "NotifyBucketChangeCommand(" << getBucketId() << ", ";
-    _info.print(out, verbose, indent);
+    out << _info.toString();
```

Comment:
Could be simplified down to just `out << _info;` (unless we want stringification to be explicit)

Reply:
Correct, will do

Reply:
Fixed